ideas-list.md

```diff
@@ -0,0 +1,82 @@
+# 💡 Google Summer of Code - Project Ideas List  
+
+Welcome to the **Google Summer of Code (GSoC) Project Ideas List** for **WSO2**! 🚀  
+
+📌 **Before applying, students should:**  
+- Read the [official GSoC guidelines](https://google.github.io/gsocguides/student/) on selecting a project.  
+- Choose an idea that aligns with their skills and interests.  
+- Reach out to mentors for guidance before submitting a proposal.  
+
+---
+
+## 📂 Project Ideas  
+
+### Summary of Projects
+
+Identity and Access Management:
+- OpenID Connect Client Initiated Backchannel Authentication (CIBA) SDK with Python
+- Financial-Grade API (FAPI) SDK
+
+### 🚀 1. **OpenID Connect Client Initiated Backchannel Authentication (CIBA) SDK with Python** 
+
+**📅 Project Duration:** Long (~350 hours)  
+**🔥 Difficulty Level:** Easy  
+**👨‍🏫 Mentors:** Thilina Shashimal Senarath (thilinas@wso2.com), Omindu Rathnaweera (omindu@wso2.com)
+
+**📍 Description:** 
+This project aims to develop a Python SDK that simplifies the implementation of the Client-Initiated Backchannel Authentication (CIBA) flow, as defined in the OpenID Connect (OIDC) specification. CIBA enables authentication processes where the end user interacts with a dedicated authentication device, separate from the client application, making it ideal for scenarios where the client device lacks a direct, secure interface for user interactions.
+
+The SDK will abstract the complexities of interfacing with identity providers (IdPs) that support CIBA, including **WSO2 Identity Server** (WSO2 IS). WSO2 IS is an open-source identity and access management (IAM) solution designed to provide seamless authentication, authorization, and identity federation capabilities. By offering a streamlined, easy-to-integrate solution for initiating and managing backchannel authentication requests, the SDK ensures interoperability with any compliant IdP while enhancing overall security and usability.
+
+Note that familiarity with the specification is not a prerequisite for the project.
+
+**🛠 Possible Technologies involved: :** 
+- Python
+- OAuth/OIDC protocols
+- Identity and Access Management
+
+**📚 Learning Materials:**
+- https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0-final.html
+- https://ob.docs.wso2.com/en/latest/learn/ciba/	
+- https://ob.docs.wso2.com/en/latest/develop/mobile-application-for-ciba/ 
+
+**💪 Skills / Technologies expected for the project:**
```

Comment:
Shall we say "Expected skills and technologies for the project"

Reply:
Fixed with [684c4b6](https://github.com/wso2/google-summer-of-code/pull/1/commits/684c4b65bfad8d9231190d8c4f3a967496ff3401)